azure-pipelines-dependency-graph-testing.yml

```diff
@@ -245,13 +245,23 @@ stages:
 
           - script: |
               # Install dependencies
-              pip install pytest-cov pytest-icdiff pytest-mock pytest-azurepipelines invoke
+              pip install -r requirements-dev-test.txt
 
               # Run unit-tests
               invoke tests --ci --cloud
 
             displayName: 'Unit Tests'
 
+          - script: |
+              pip install coveralls
+              export CIRCLE_BRANCH=$BUILD_SOURCEBRANCH
+              coveralls
+            displayName: 'coveralls'
+            env:
+              CIRCLECI: 1
+              CIRCLE_BUILD_NUM: $(Build.BuildNumber)
+              COVERALLS_REPO_TOKEN: $(common_secrets.coveralls)
```

Comment:
The `coveralls` packages takes care of pulling the `COVERALLS_REPO_TOKEN` from the environment.
Need to verify that this is the correct way to pull values from variable groups.

Also, it's possible this is correct but that it being secret is causing issues? See the note below.

>As per [#245](https://github.com/TheKevJames/coveralls-python/issues/245), our users suggest leaving “keep this value secret” unchecked – this may be secure enough as-is, in that a user making a PR cannot access this variable.